common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -153,8 +153,12 @@ private static CommandResult executeCommand(BaseCommand command) {
         }
 
         if (baseException != null) {
-            //Post On Error
-            commandResult = new CommandResult(CommandResult.ResultStatus.ERROR, baseException);
+            if (baseException instanceof UserCancelException) {
+                commandResult = new CommandResult(CommandResult.ResultStatus.CANCEL, null);
```

Comment:
Shouldn't this be `new CommandResult(CommandResult.ResultStatus.CANCEL, baseException)` ?

Reply:
@kreedula It wouldn't make a difference since in the case of cancel, the result object isn't used because onlike the onSuccess and onError, cancel callback does not accept any parameter such as a result value or exception as is the case with onSuccess and onError. See this: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/dev/common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandCallback.java#L26